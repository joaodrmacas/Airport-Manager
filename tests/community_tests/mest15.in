a LIS Portugal Lisbon
a OPO Portugal Porto
v TP1 LIS OPO 23-03-2022 14:10 00:50 15
p LIS
q
