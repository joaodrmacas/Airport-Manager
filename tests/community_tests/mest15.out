airport LIS
airport OPO
TP1 OPO 23-03-2022 14:10
